shpkpr/marathon/schema/app.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
I was going to be extremely impressed that you had knocked out a schema this big that quickly :smile: